### DISCUSSION TITLE
My Suggestions list

### DISCUSSION THREAD

**Discussion by commentator24** [ValveSoftware/Dota2-Gameplay#3518] [Ideas]
Thought might be nice if I share my ideas here. ty if can upvote
Also looking for any work / would love to be more involved with dota / any custom game development or github triage / moderation role to close issues that are not related to bugs / dota 😊

__Cosmetics / skins__
- [ ] Feature request portal / centralised community feedback system. e.g. Vote / buyers interest indicator to show the level of demands of specific feature / skins suggested? encourage licensing partnership with commissions or crowdfunding to speed up creation of many personas and taunts? e.g. japanese animators pudge hook + one piece / darkseer + one punch man. maybe best story competition as well?

__Steam / dota profile / mini games__
- [ ] Mouse drag to highlight text chat in Party / Post Game tab. was removed in 2020? Maybe nice if separate history of ally+all chat is created in main menu tab like Party / Post Game tab
- [ ] Main Menu UI reworked. too many search boxes?
e.g. Watch / Replays / Search game ID or Downloads. 
e.g. Caravan check duplicates have to click Armory / Search armory items. 
Might be nice to have a dropdown search armory / game ID like search friend ID section at top right. or ability to customise dropdown at top right like mini profile edit
- [x] Re-bundle + right click option to Use with..(choose parts to be rebundled) + gift non-tradable duplicates to friends e.g. accidentally unbundle duplicates / prefer to use friend gifted duplicate set that looks nicer with custom gifted message + re-bundle other duplicate sets by right click option
- [ ] Request autograph / custom message or add sticker to item (like choose friend to gift+message, but only request to sign / add message. e.g. happy bday msg from multiple friends or add image logo stickers / sponsors)
- [ ] Daily/Weekly steam/dota crafting system/materials like typical mobile games? e.g. some duplicate items cannot recycle / gifted / traded. Right click option to combine duplicates? e.g. combine 2 same item to get lvl 2 item level. Item level will be tracked to unlock custom colour scheme style / custom stats / effects like gems / play hero voice lines. Maybe players can edit it too like creating effigy / workshop tool. in path exile there are item levels, tiers of effects, and many item mechanics like corrupting them for unpredictable outcomes or use crafting/vendor recipes to purify. maybe similar concept can be used to combine same 5 items to gain random stats with slightly better result +1 or yield item xp based on rarity / quality.
e.g. craft / draw stickers that tracks certain stats to earn materials / shards.
e.g. craft friendship gems that tracks winning streak / number of victories predictions with chosen friend.
e.g. craft custom relics for dota items / sockets / custom challenges e.g couriers killed / guild contracts / track counter of completed challenges / for use in game. Socketed gems reward shards upon level up for milestones in stat tracking, or maybe to craft double mmr tokens etc. Incentive to use artificer hammer and make sockets.
- [ ] Hide dota profile level #9143
- [ ] Dotaplus Profile rank change column shown by default
- [ ] Dota fantasy territory war #5522 Would also be very nice if devs can implement my fantasy caverncrawl similar to planetside2 / battlefield.
ps2 has resources at the top and nice UI that flips the colour of connected lines (factions). maybe unused crownfall tokens can be resources?
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/109324155/7af3ed69-78db-4b1e-943f-22a692a04c7d)
bf has sectors that unlock after certain milestones.
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/109324155/e5fb44dc-11b9-4e88-976f-f21d7ad71b3e)
cavern crawl can utilise similar concepts / sectors e.g. fires of ulmarain / chasms of blueheart
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/109324155/a0c85803-a110-4863-8484-41a9002fd0ed)
crownfall has tokens, nodes and finish line.
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/109324155/182be5b4-ff33-47fa-8833-29be96424cf4)
- [ ] In PS2, 3 empires fight for influence to gain more rewards / resources. Similarly, cavern crawl or crownfall designs can be utilised to add fantasy endless war elements where guilds fight for guild influence / factions. Players can click on the map to select the node / spawn / "help" fight there and must capture regions / bases (nodes) along owned connected lines. e.g. 4 world factions: str / agi / int / universal heroes played in ranked mode. To flip factions, simply win as str/agi/int/universal for that node you selected. Scores update once an hour.
Further development suggestions:
- [ ] 2-3 fantasy global (or World) map as sectors to separate rank / turbo / ability draft maybe nice.
- [ ] Fantasy region (or Territory) map for each server e.g. sea / australia to show which region and factions are dominating in the world map. Similar to cavern crawl goal to get 3 item sets, these could be 4 faction base str/agi/int/universal starting points. Basically a fancy way to visualise dotaplus global trends / meta in each region.
- [ ] Fantasy guild / node map for each node in the region map. Like crownfall nodes, but can click to see which guild is very active on that node. each node with highest seasonal guild activity will show guild icons / flag. A fancy way to visualise guild rankings. Then maybe add few mechanics e.g. can be taken over by other guilds once a week / spend guild points or tokens to "upgrade" nodes / bases - build custom map paths / defenses to stop the node being taken over by other guilds easily. e.g. once each day/week guild can build up their own army defenses / fantasy cards team. more faction points contribution = more bonuses for fantasy army / cards in that node, but weaken other types / effects / connected adjacent nodes to prevent abuse? e.g. upgraded nodes allow only guild / allied guilds to edit node profile to design custom "Node map" like mini cavern crawl connected lines concept / set traps / towers along connected lines for guild battles / defenses. max showcase budget increase as guild gets bigger / gain fame? e.g. in nodes minimap instead of item sets to claim we could have 4 starting factions / 3 random active guilds placed each week, like battlecup. Guild with most connected lines win, then next week randomised again vs less active guild if lose. Perhaps some nodes require specific goals like winning battlecup tier 3-8. It'll be nice to see which guild win the most battlecup for each tier, or perhaps encourage social interaction like winning with allied guildmates / party / Add new friend as helper / coach.
- [ ] Guild / Faction points earned for each node. The more we play str/agi/int/universal heroes on selected nodes and win, the greater the influence / connected lines. Spend tokens to build specific influence / connected lines?
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/109324155/4a02569a-3a40-41b1-8d2d-d789baecf88f)
- [ ] up to 10 guild alliances to maintain connected lines e.g. has same goals playing as batrider / int heroes to boost certain faction points or add alliance.. allow guild alliance quest to do guild challenges together when 5 player partied.
- [ ] maybe better if we can create and place our own fantasy cards / MVP to highlight our hero strengths / bonuses (just like mvp cards) to earn points for that node map. e.g. each season (3 months?) we have our unique player card that starts with 40 HP (or 40 games). we can level up our player card and help join fights to earn points or MVP tokens for that region until we lose all our HP (lose 40 games). Regain 1HP each day. devs can implement elements from artifact card game e.g. everyday (24h cooldown) we have 1 action phase to use spells / items / abilities to buff units / hurt opponent guilds / move player cards to a better position / stay out of battles from that cavern crawl map.
- [ ] future development could be something like addition of different fantasy races / base types / turn based RPG style to get unique mercenaries / army / inventory / armory items and provide unique bonuses, e,g, mercenary / hero cards to help your team / guild dominate the fantasy lanes (connected lines like cavern crawl) and capture more region or even customised lane creeps? At the end of each season guild influence tallied and reset. Resources could be crafting / reroll materials for candy sacks as implemented during battlepass.
- [ ] End goal maybe community rewards such as certain heroes slightly buffed / nerfed. or allow community votes for buff / nerf option suggestions at the end of each season (3 months?). e.g. if trends is below 50% winrate or 1 million total wins as batrider not achieved we dont get the buffs/nerfs. Hero patches to nerf/buff specific faction / heroes? or make predictions like battlepass e.g. most successful hero, most kills, most banned in the region of server.

__Rank system__
- [ ] 5 MMR for each roles pos 1 2 3 4 5
- [x] separate MMR for each hero. Maybe bonus MMR for winning streaks as same heroes? I see MMR delta section in dotaplus. maybe this could be implemented soon?
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/109324155/7ced17a1-2ed0-4ba1-a450-7e71a3d5595f)
- [ ] reset MMR / rank confidence % every season + seasonal treasure / shards / trophy? Incentive to play during certain months.
- [ ] more MMR with win / lose streaks / MVP / performance. e.g. +15 mmr for MVP / +5 honorable mention / +5-10 performance. or perhaps rewarded with guild points / shards. 3 win streak +30 (60 mmr), 6 wins +60 (90 mmr) so smurfs get to their proper MMR faster. Also fun to try keep win streaks.
- [ ] MMR for normal all pick / Captains mode / turbo with tick box option to update MMR or not (like the old +5 or -45). this way can sort of party rank without MMR too far apart message, and players can decide to play for fun / competitive in normal matches for less / half MMR but risk normal loss amount. e.g. +10 -25.

__Map / Mode__
- [ ] Ranked all pick reworked like tournament style Captain's draft without captains. Fastest player to click choose ban during ban phase. Players pick their own hero. Pick phase happens at the same time for both radiant and dire instead of taking turns e.g. first 2 hero picks, then 3 bans, then 2 picks, then 3 bans, then 1 pick (if both team choose same hero, it will be extra ban / ban slot used up until 3rd time happens. then fastest player gets hero as usual).
- [ ] New selection mode: ticking turbo + other modes like all random will create a turbo version of that mode. turbo ability draft!
- [ ] Turbo: allow couriers to be killed for guild contracts. 2x faster courier respawn / 2x faster death respawn time
- [ ] New unranked modes: Ultimate Draft / Aghanim / Crownfall / Gold Rush / Hardcore / Insane / Immortal / Mirror / Stackable
- [ ] ultimate draft / aghanim: similar to ability draft + aghs lab style skill upgrades (where player given 3 choices to upgrade). Take turns to pick heroes like CM mode and then ultimates. Can pick same heroes, but ultimates are drafted in ability draft style, so even with 5 zeus, only 1 zeus ultimate is available to be drafted and player take turns to pick ultimates and choose the shard upgrade.
- [ ] Crownfall: small buffs when drafting heroes with similar tokens e.g. Polearm + 25 attack range. or maybe extra +2 stats
- [ ] Domination / Sacrifice: Highest Kill after 30 min win / Every 10 min year beast is summoned to help destroy throne.
- [ ] Gold Rush: players die without losing gold. all items have sharable recipe (can help other finish items e.g. buy midas recipe for others). Maybe some items become cheaper/expensive if team already has it. e.g. first midas 1750g. 2nd 1500g, 3rd 1250g. This could make interesting gameplay e.g. someone richer can help finish your item, instead of waiting for 20 min midas. and make responsibility fall on team decisions e.g. 1 midas not that useful, 5 midas very effective?
- [ ] Hardcore: Friendly fire is on. e.g. jug spin can damage allies. Able to steal enemy shop limited stock items e.g. smoke / wards. e.g. run out of ward stocks and start cutting off enemy supply.
- [ ] Insane: HP bars on creeps / enemy heroes no longer visible. 
- [ ] Immortal: free buyback gold cost / halved buyback CD? receive Weakened debuff hp reduced by 20% for death duration or maybe cheaper cost for every buyback etc. skills also halved CD or instant. 
- [ ] Min (Minimum Skills): Start with all skills level 1 ready to level
- [ ] Max (Max / Unlimited Level): Start at max talent / skills. Only gain stats per level
- [ ] Mirror (Same hero): can pick 2 techie 3 tinker etc.
- [ ] Stackable: allow items to have stacking effects at higher prices / cooldown. e.g. 2 cuirass 2 deso 2 greaves. first one 1300g recipe, 2nd one is 2600g if team wants to buy second one. or maybe allow 2nd one after 7 min cd? or time locked like shard / limited stock 1 for team so makes team more aware if someone already has it.
- [ ] 4 Seasonal map changes every 3 months e.g.
Autumn less trees placements on edges of map, or some trees automatically wither, allowing players to tunnel behind enemy lanes easily. -100 day vision, +100 night vision. limited stock iron branches to plant trees that last 5 min.
Winter -200 day -200 night vision / tower vision, winter debuff e.g. -20% movement -20% health regen when standing still for 6 sec, but gains armor per second up to +10 after 15 seconds not moving / seasonal shop items e.g. snowball slow enemy for 2 sec.
Spring more water areas as snow melts? river buff +20% move / 20% regen faster when standing on rivers like ocean heart effect.
Summer +200 day + 200 night vision but also more trees to block vision, faster trees respawn. Summer skills e.g. all players can do lv 1 sunstrike 25 sec cd, 125 dmg. Or maybe summer buff / debuff / cached weapons (duration / consumable based inventory) e.g. +5 regen or able to destroy trees like batrider for 8 seconds /  stackable burn dmg, like huskar lv 1 burning spear when taking player damage / gain tree-walking / destroy trees like batrider for 8 seconds. this will make players really feel seasonal terrain effects and different seasonal playstyle.
- [ ] Gold gathered by radiant / dire can be used to upgrade siege units / barracks / buy special "ancient" units / skill at Throne or effigy. e.g. eagle to scout an area or upgrade siege units (time locked like neutral items) e.g. catapult > catapult skill like snapfire mortimer kiss > tank / special unit races 30 min+ only > year beast 60 min+ only.
- [ ] Introduce races to customize lane creeps / buildings. Players can enter game as 1 of 10 races...human, machines, beasts, Dark Reef/Naga, orcs/trolls, undead, high elves, dwarves, titans, and skywrath with unique skills / weaknesses. Player effigies will be the control point to command unit formations / behaviour / building tech. Each player own 1 uncontrollable lane melee creep. Or maybe coaches can enter as the strategist role
- [ ] Introduce Rune shop: power up scrolls? dd etc 500g? area healing / area armor / area teleport etc.
- [ ] Shop / hero guide reworked to show 5 items instead of 4 #9824
- [ ] Shop / hero guide Item grid: similar to heroes grid, maybe nice to have 6 items grid layout when opening shop and trade window like path exile #9824
- [ ] Effigy destroyed will spawn uncontrollable immortal ancient ancestor that cannot die / reincarnate to help defend throne. lane creeps ignore these units to attack throne. gold gathered by radiant / dire can be used to upgrade your "ancient" e.g. spend gold to buy "fear" skill, CD 3 min. 
- [ ] Effigy gather souls of the dead, when destroyed, you can buy faster death respawn timer / cheaper buyback cost, depending on how many souls collected / how many times you bought back. This means losing team has more chance to comeback if their effigy destroyed.
- [ ] Mini game during death? e.g. send courier shovel death location to make buyback cheaper / faster (increase reliable gold counter, not actual gold). 

__Gameplay__
- [ ] All players gets 3 new abilities to climb cliffs + cut tree + plant tree last 20 sec / 20 sec cd like tumbler's toy so players dont get stuck by unfair pudge hooks / magnus / rubick lift. Could make interesting gameplay like jumping into trees to hide then tp, while enemies also can cut tree without quelling blade filling up important item slots.
- [ ] All items can be disassembled at any time (buyer only for rapier, and cant be disassembled if player damage taken the past 13 sec e.g. for rapier abuse).
- [ ] Portals at outpost offer 1-way free teleport back to base after 6 seconds channeling, like underlord ult. Gives new reason to capture outpost.
- [ ] Ctrl + click inventory to sell items quickly. Selling is 100% value, but limited stock for team (has 7 min CD)?
- [ ] Ctrl + click ability to use alt skills version e.g. allow Ctrl as alt modifier in setting.
- [ ] Dota labs suggestions for Smart AI, custom healthbar slider for axe ult, custom setting to save 1-3 drawings / overlay and show to teammates. e.g. overlay 1 has all suggestive ward locations during laning phase. overlay 2 is for defensive ward locations / areas to avoid fights. overlay 3 for offensive wards or where to fight/cut lane during smoke or hide during T3 push.
custom setting to View / hide each ally/teammates ping/drawings. 
toggle minimap overlays to show teammates drawings, tooltips and 6 right click abilities modifier e.g. self cast, autocast, cast to allies only / cast to enemies #15868
- [ ] Couriers reworked? no more ancient type unit. Casting spell instantly kills it / deal 1 dmg.
- [ ] Move while channeling. Similar to puck leash, if you move too far while channeling, the casting will cancel.
- [ ] Mounts 4 types like in diretide?
- [ ] Runes spawn both sides of river. Or one bounty rune? Not fair one side gets nothing. Duration halved but duration stacks. e.g. picking up 2 haste rune = 11s + 11s = 22 sec. 
- [ ] Rune: double damage - shortened 22s duration like haste. 45s unfair advantage to roshan / teamfight.
- [ ] Rune: haste unlock max ms, but only +50% ms. might be super fun to see.
- [ ] Rune: Elemental? fire water earth wood and metal (gold)?
fire like lina fiery soul stacks faster attack speed / ms / Flat -1 sec CD reduction? takes damage every time skill used.
water HP heal or gain charges every time skill used like wand. heals at the end like regeneration rune. or just a simple 60 HP/MP heal?
earth more armor every time attacked like timber passive?
metal (gold) more gold like alchemist passive? etc.
- [ ] Rune: Power down. might be interesting to have power down runes or random positive and negative effects at same time. e.g. regen to full hp then poisoned 300 dmg. double dmg but increased dmg taken. illusion but reduced dmg, hasted but then slowed 20% for 6 sec at the end of effect. 
- [ ] Runes bottled can be thrown at enemies / allies?
- [ ] Siege units mountable, can click to enter and hide but not control. Will be interesting to see 5 players hide in siege unit and surprise enemy.
- [ ] Tower levels. Neutral creeps abilities level up at min 15 / 30, how about tower as well?
- [ ] Throne has global skill similar to snapfire aghs gobble up to shoot allies where help is needed. 3 min CD. can make interesting gameplay e.g. tp back to base then gobble.
- [ ] Talents +2 attributes has negligible effect. how about +3 or +4 per level?
- [ ] Talents / skills undo at any time / pay 250g at shop to reset?
- [ ] Watchers: allow any unit to capture this e.g. dominator / LD bear.

__Heroes__
- [ ] All heroes 4 normal / shard skill. Upgraded Aghs is like switchable tread for 3 upgrade options (like in aghs lab). Consume to use.
- [ ] Ability to cancel skill early / use alt version: Dawn(2), Primal(2). e.g. maybe alt click celestial hammer allow hammer to return early. Not fair disarmed and had to wait until duration over.
- [ ] AM 3 > counterspell reworked like lotus (not fair block at 3 cd). adds 45% magic resistance while active.
- [ ] AA ult > controllable pointer like elder titan skill 2 Astral spirit returning to hero. Allows change direction to desired location if fast enough to move it around. pointer does not grant vision
- [ ] Centaur Leap Stomp: how about leap like mirana then stomp? or has 4 charges for each hoof (1/4 effect)
- [ ] Centaur Hitch a ride reworked like bloodseeker aghs. can on and off anytime, but centaur takes 7% HP dmg per second. non lethal. Cannot be turned off while on CD 8 seconds.
- [ ] Darkseer normal punch reworked as active ability called surged punch. unlock max movement speed for 6 sec duration + next attack normal punch / vector targeting to knockback enemies into wall / autocast option to uppercut like walrus punch.
- [ ] Doom immune to doom. the damage heals him if reflected / aghs self cast. still silenced / cannot use items. would be nice if get increased ms / attack speed too like mask madness or nightstalker ult, to show that this is doom enraged version
- [ ] Earth spirit / Tusk controllable W skill similar to primal beast charge to change direction (but cannot stop at any time). even better earth spirit / tusk can toggle on and off like rot, damage themselves too like ruptured.
- [ ] Gyro Wingover: how about using real combat maneuvers? similar to icarus dive + tiny toss, click a spot on the ground to dive down and toss plane up and down before returning to original position. during toss it provides time to use skills.
- [ ] Lion finger bonus dmg also adds 25 mana cost for balance. Right click to stop adding bonus if cost too much
- [ ] Lycan summon limit removed
- [ ] MIrana solar flare change night to day
- [ ] Phoenix skills / Invoker sunstrike / zeus nimbus make day effect on certain spots to weaken nightstalker / other night heroes.
- [ ] Rubick aghs ult right click buff icon to remove last stolen spell. Or ability to swap hotkeys might be nice.
- [ ] Shadow Shaman deny full hp wards with other wards like clock cog
- [ ] Storm talent 25 + Facet 2 also move to target position
- [ ] Tusk walrus kick or shard creates immovable frozen sigil at landing / casting location? to reuse sigil.
- [ ] Tusk walrus punch becomes vector targeting and also knockback enemies for a shorter range than walrus kick.
- [ ] WK lifesteal aura applies to helm dominator creeps.
- [ ] Zeus - Movable nimbus 100 speed. Also interact with knockback spells e.g. drow gust or create sladar water puddle?

__Items__
Consumables:
- [ ] Area clarity / flask.
- [ ] Dust cheaper 50g not 80g. Cause 30% blind instead of 20% slow, and provides flying vision over an area and deal 1 dmg every 3 seconds to prevent blink dagger that last 6 seconds. Wards can be revealed/damaged -1hp in this way, CD 15 sec. will be interesting to blind enemies / prevent blink dagger / easy dewarding by simply casting dust on cliffs. Isnt it unfair for melee heroes having to go around cliffs to deward?
- [ ] Dust + Blood grenades can be combined to apply blind + slow and reveal in huge aoe effect.
- [ ] Dust + Faerie fire can be combined to grant flying movement for 6 seconds.
- [ ] Faerie fire, blood grenade max stack 3 like other consumables.
- [ ] Flare +100 vision radius. 50g: like clock L1 rocket flare, reveal location for 6 sec.
- [ ] 3 Healing lotus combine reworked. Use up all charges when used. a bit annoying that it use up 2 item slots. Or only auto combine if double clicked. click on hero to use.
- [ ] Greater faerie fire 1400g. +40 dmg. 450hp when consumed. only available after 15 min like shard and can be placed in neut item slot.
- [ ] Hookchain 100g, +0.5 armor, max 3 charges. 4s cd. similar to timber chain lv 1, hook to any trees 600 units away and deal 70 dmg to any units in path while pulling the player towards tree. when charges used up, the item disappear. this could make interesting and aggressive early gameplay.
- [ ] Ironbranch limited stock like sentries. planted trees last 5 min. can make interesting gameplay like plant tree on cliffs to prevent players ward that area.
- [ ] Snowball. Seasonal item? 0.5 sec slow enemy CD 15 sec.
- [ ] Smoke / sentries / consumables free 0 gold. limited stock anyways.  No more excuses not to buy.
- [ ] Smoke from couriers (or any other heroes) apply the effect to any allies that come within range.
- [ ] Wards expired visible for the rest of the game to show previously placed locations.

__Etc:__
- [ ] Battlefury half effect for ranged heroes.
- [ ] Bottle stackable charges when used (up to 3 stacks). greater effect or extended duration. Reduced healing 110 > 60 hp 60 mp.
- [ ] Teleport slot is upgradable 5x like teleport boot upgrade to lower CD up to 40 sec and +40 faster MS (+5 ms and 8 sec per level) Players don't have to sell their boots / buy 2 boots anymore.
- [ ] Glassboots / runeboots. Players usually sell their bottles mid game. how about bottle + boots?
- [ ] Phaseboots able to phase trees / cliffs with very slow movement (100) e.g. movement debuff across cliffs / trees.
- [ ] Waterboots hp regen 10 HP regen and 5 mana regen while in river like ocean heart.
- [ ] Jetboots. Leaps the player forward really fast like slark pounce! Has 2 charges. or maybe behaves like clockwork jetpack for flying movement. obviously should cost about the same as slark aghs 4200g. Will make interesting play for centaur hitch-a-ride, batrider etc.
- [ ] Echosabre half effect for ranged heroes (-50% dmg or 12 sec cd?).
- [ ] Escape items upgradeable to provide basic / strong dispel / 1-5x effect. e.g. force staff 1-5.
- [ ] Helm dominator etc give creep bounty gold/xp when used.
- [ ] Midas get +1 level of neutral items. CD 360/270/180/90 seconds? 5 levels like dagon. grants Greevil's greed at level 5 like alchemist ability.
- [ ] Neutral items get stronger over time. e.g. has tier 1-5 effect like dagon.
- [ ] Ristul Ring / spike ring. like soul ring but require 2 slippers. consume 170 hp to give 40 dmg for 10 seconds like dagger of ristul.
- [ ] Essence Ring. like soul ring but require 2 mantles. consume 170 mana to give 400 hp for 10 seconds. Like Essence ring neut item.
- [ ] Shadowblade / silveredge reworked to apply separate bonus physical dmg buff if the invisibility accidentally broken e.g. cast skill during invis, lost movement speed and invis but still can apply bonus dmg within duration left.

__Pre game (drafting / profile)__
- [ ] better MVP feature to detect smurfs / win streak. Just like winning battlecup, if player has 3 win streak or more in recent 15 matches, player profile features MVP streaks for 3 MVP heroes used most. This way opponents can plan ahead / counter draft.
- [ ] Swap role feature. Bring back 5 man party role for dotaplus quest progress.
- [ ] Heroes tab level slider to see changes default base HP at level 1 to 30. 
__Post game__
- [ ] More play style in this match stats, similar to Fantasy card system. MVP system should offer similar details, so players can analyse ways to improve and get MVP, such as in stacking and pulling, harass in laning phase (first 10 min), comeback teamfight assists etc.
- [ ] New breakdowns (dotaplus): damage tab also show timeline for total damage done vs time. Hover mouse to show death details + damage by who. Perhaps could help identify griefing behaviours e.g. afk farming 0 dmg teamfights / feeding kills. 
- [ ] New graphs - damage / deaths to show player / courier deaths and damage to other heroes. Similar style as player item graphs to hover mouse and see time + killed by who.
- [ ] New overview minimap: that shows player locations over time (with slider bar). this can be used as replacement to player deaths graph as player location is reset to base when respawning. Will be very useful to briefly review what went wrong without downloading the replay. And perhaps to report afk / feeding / griefing players. 
- [ ] Scoreboard: better tracking of total gold earned for team e.g. picking up bounty runes / total gold earned from courier kills / dewards.
 
__Rewards / Additional features__
- [ ] The International for developers. Event for future dota 2 development and showcase your ideas, e.g. New heroes, skill effects, item sets, themes, story, AI etc. Why only pro players get prize money? I think developers should be given chance by community vote for features we'd like to see (like arcana voting). Proposed ideas will remain until it is forgotten or received too many negative feedbacks from community. Mini prize pool for each bug fixes etc.
- [ ] Battle stats (dotaplus) tracks courier kills and bounty runes taken.
- [ ] Battle stats (dotaplus) not updating? Default columns needs a lot of rework too #9520
- [ ] Battlecup: tickbox to enter as team / guild
- [ ] Battlecup: Spend 10k shards to buy weekly battlecup ticket for a friend / join for free.
- [ ] Battlecup / Guild: Spend 20k shards to host/join regional weekly guild / battlecup / tournaments / create friends of friends seasonal league for fun or as a way for community to regulate itself. free battlecup ticket for yourself / your team. Challenge others / bragging rights for weekly battlecup champion. Possibly recognised as chances for entry to DPC / TI for each Tier 3-8? Or create guild leaderboard.
- [ ] Battlecup semifinal go to next round e.g. grandfinal instead of refund. #9175
maybe need a global counter to count total teams participating in the regional battlecup for each tier. Just like how currently we can see approximate global roshan kills / watcher activated etc.
Lose quarterfinal = participation 500-1000 shards. incentive for dotaplus members to play. a lot of my friends say if don't win 3x = just wasting time. might as well ranked. 500-1000 because perhaps might get team who forfeit at the start of game. e.g. 500 if forfeit...1000 until throne destroyed. but doesn't have to be complicated like this.
Win quarterfinal + cant find = 2500-5000 shards + trophy win OR auto find grand final opponents. please no refund.. only refund if coordinator down or minimum players not met regionally...
Win semifinal + can't find = 5000-7000 shards + trophy win? 7000 because usually distribution is 36% less than grand prize maybe
Win grandfinal = 20000 shards + trophy win as usual.
- [ ] Chatwheel has Dive! but not Don't dive! Would totally spam don't dive every game.
- [ ] Chatwheel allows select ally + phrases. e.g. when HUD is selected on ally, item suggestions alt ctrl shop item etc focused on that player e.g. "I think someone (purple) should buy lotus" e.g. get back! becomes (purple) get back! this would be clearer than spamming chatwheel to get back / bait / and 3 players buy lotus.
- [ ] Effigy post custom message to all chat when destroyed. Players rarely check out effigy and this will bring their attention to it.
- [ ] Fantasy cards: Create our own fantasy card. 
- [ ] Fantasy cards: recycle silver/gold cards can be time consuming to keep clicking yes..imagine clicking 100.. #9857
- [ ] Guild contracts 3 stars x10 only. perhaps new contracts style? e.g. win 3 turbo to get 3 star #5522 
- [ ] Guild influence leaderboard: similar to DPC standings tracker, has each division e.g. herald-immortal / can add up to 10 teams for guild alliances. Weekly guild vs guild events like battlecup (but represent guilds / tier divisions). Guild tracks number of battlecup won by guild and show custom display when you check out player guilds during game (e.g. like winning battlecup) during the season for bragging? Or seasonal guild battle reports for more shards. Guild points may be used for bonus shards earnings / standing. e.g. Tier 3 bonus 3000 shards / -3000 guild points for guild standing / Spend guild resource / points to gain seasonal / craft consumable party effects like battlepass fountain / teleport effect level 1, balloon charges etc.
- [ ] MVP: Reveal reveal MVP calculation please - seems broken #8438
- [ ] Overwatch / coaching reward 50 shards for positive feedback.
- [ ] Relics: Grandmaster tier reworked. 8000 reset to 0 and reward 5000 shards. incentive to keep playing same hero. Or infinite tracking of levels e.g. grandmaster 1....300. Would look fearsome to see the numbers.
- [ ] Seasonal MMR reward shards e.g. herald 5000 / guardian 10000 / crusader 15000 and so on. 
- [ ] Scoreboard show last known (not current) inventories for both radiant and dire, including couriers (instead of only allies). So a good captain who check teammates items can plan ahead e.g. axe blink in courier..go smoke gank soon. i had games where teammates said bkb in courier after teamfight over. no one check their courier items.
- [ ] new hotkeys / click to select enemy 1-5 as well.  e.g. press x1 is like clicking 1x at top bar. press x2 centre camera to last known / clicked / pinged position (if fogged). this way teammates not confused where the enemy is.

__Behaviour score__
- [ ] Behaviour score reworked like shard currency / MMR. Or implement behaviour score log e.g. shard currency merged with behaviour score. lose 10k+ shards for bad behaviour e.g. dc or profanity / -ve amount auto LP. Players will behave nicer if they can lose shards for bad behaviour.
- [ ] Chat history to teammates saved as public, like All chat. enemy just cant see them during game as if muted text
- [ ] Mute toggle all team chat can also toggle to temporarily unmute all
- [ ] Max 15000 to reflect 15 games. or 10k for 10 games.
- [ ] -1000 behaviour per game / communication abuse reports. reporter also get -250 to prevent abuse. weekly max report removed. 1000 should be fair so 4 reports 4000 can almost you into low prio at 3000 behaviour.
- [ ] -250 behaviour per pause / unpause. Max 3x and 30 sec wait removed. This will make players think twice about pausing / unpausing e.g. some players pause then type ? after a kill.
- [ ] -50 each death. e.g. 10 deaths -500. reports for feeding no longer needed.
- [ ] +50 overwatch reviews / commends / coaching.
- [ ] +1000 playing a game without any reports
- [ ] new report reason: Passive movement / casts / AFK e.g. teammates afk farm / never join teamfights / pretend to push towers / rat / not defending base. i think it's not fair for struggling teammates to wait 5 min until abandon but the afk player simply gain xp once every 5 min / hide in trees. With this new report, secret timer for not joining teamfight starts after the report. Auto abandon if does not do player damage within 3 min. Can report again to reset the timer. 
- [ ] Able to report players when watching replays / adjust overwatch ! mark to look for suspicious mouse movements / casting / scripting / hacks. Replays can add / remove ! mark on time to note at which point the player is griefing / scripting for Overwatch review. Someday help AI figure out when griefing occurs.
- [ ] Able to see total likes / dislikes we give on each player. Avoid list notes should not disappear and kept personally 
if I have more ideas i'll edit here. hope you like my ideas and comment below

**Comment by DavinS1** [comment]
A really good post, very thankful and hopeful that you will write many more posts like this one.


[My Milestone Card
](https://www.mymilestonecard.bid/)

**Comment by ghost** [comment]
Interesting topic to talk about :)

**Comment by Homilos** [comment]
When developing software or a finished product, it is worth paying detailed attention to usability testing in order to evaluate your product. For this reason I prefer [usability testing service](https://qawerk.com/process/usability-testing/) because they can provide an excellent opportunity to test your product and make it fit your expectations. I think it's perfect for such cases as a sign of the quality of your work.

**Comment by Eaioumore** [comment]
Hey.. thank you for contacting me on my thread!! And the recommendations for Dota Arcade!!

I'm wondering if your post got you in contact with a developer??

I hope you are successful!

**Comment by nyczalex** [comment]
I scrolled through it and I really like and agree with almost all the rewards/additional features.

As for the behavior score suggestions, I think the easiest and best way to handle it is just calibrate it based on amount of games played and your commends/report forth on. The more games you play play, the easier it is to raise it because you shouldn't be getting massive reports every game, especially if you cannot chat/ping/etc. If you do for some reason, that's on you. Everyone has bad games so 2 games of someone feeding and getting massive reports should in no way be more effective then if you have 10 other games full of commends. The whole point of the behavior system is to revoke the privileges of using chat/ping/etc. which helps supposedly higher chance to win games and get better. At the moment, it is not at all like this.
People abuse the system, why is there even report options  if you can false report someone then make it so difficulty to get out of it. Then penalize ever harder when it happens again? Again, all those things are a privilege and key tool to increase chance of winning, its already a punishment to restrict them from using it but to do it in the "longer term" model is crazy when the report systems is more automated then ever.

**Comment by mrstealyommr** [comment]
The separate mmr per hero would be crazy to balance. Plus a lot of mechanical skill translates to most heroes just fine. If you can micro one hero you can micro them all etc. So it wouldnt make sense to see one hero at "immortal" mmr level and someone you almost never play at 1k mmr.  You'd still stomp the game from sheer knowledge and mechanics. Out farm, outplay etc.

**Comment by quordleonline** [comment]
Your idea is very perfect and detailed, try to plan and time each specific part then implement the stages, wish you success, you can refer to more details at [quordle](https://quordle-game.io/)

**Comment by herrty45** [comment]
[level devil 2](https://leveldevil2.io)  is a platform game that challenges players to a world filled with traps, puzzles, and enemies.
